spec/imsc-hrm.html

```diff
@@ -294,11 +292,16 @@ <h2>General</h2>
       <p>The Presentation Compositor SHALL start rendering E<sub>n</sub>:</p>
 
       <ul>
-        <li>at the presentation time of E<sub>0</sub> minus Initial Painting Delay (IPD), if n = 0; or</li>
-
-        <li>at the presentation time of E<sub>n-1</sub>, if n &gt; 0.</li>
+        <li>at the presentation time of E<sub>m</sub>, where E<sub>m</sub> is the closest ISD for which m&lt;n and E<sub>m-1</sub>
```

Comment:
Not sure the previous text was wrong. Is there a problem to render an empty ISD?

Reply:
In the previous text, the time available to render ISD_n was the duration of ISD_(n-1). This is a problem if ISD_(n-1)...(n-m) are all empty and the duration of ISD_(n-1), which is empty, is very short.